index.md

```diff
@@ -1,67 +1,65 @@
 ---
-title: The Mirin Template
+title: Main Page | The Mirin Template
 ---
+# NotITG
+[NotITG](https://noti.tg/) is a fork of OpenITG that is designed for creating and playing modfiles.
+
+Join the [NotITG Discord](https://noti.tg/discord) to learn more about NotITG.
+
 # Mirin Template
-[NotITG](https://noti.tg/) is a fork of OpenITG that is designed for creating and playing mod files.
 
-[The Mirin Template](https://www.github.com/XeroOl/notitg-mirin) provides tools to allow mod file creators to implement their ideas.
+[The Mirin Template](https://www.github.com/XeroOl/notitg-mirin) provides tools to allow mod file creators to implement their ideas. 
+It is is designed to make easing as simple as possible, while remaining incredibly powerful.
 
-Join the [NotITG Discord](https://noti.tg/discord) to learn more about NotITG.
+## Getting Started:
 
-The Mirin Template is designed to make easing as simple as possible, while remaining incredibly powerful.
+* [Choosing a template](comparison.md)
+* [How to set up the template](Getting-Started.md)
 
+## Documentation
 
-## Documentation:
-[Index](docs/index.md) - in progress
-<br>
-[Choosing a template](comparison.md)
-<br>
-[Getting Started](Getting Started.md)
-<br>
+### Mirin Template
 <div style="display:flex">
-<div style="flex:50%" markdown="1">
+<div style="flex:25%" markdown="1">
 
-* [How to set up the template](Getting Started.md)
 * [setdefault](docs/setdefault.md)
 * [set](docs/set.md)
 * [ease](docs/ease.md)
-* [add](docs/ease.md#add)
-* [for loop](docs/for.md)
-* [actors](docs/actors.md)
+  * [add](docs/ease.md#add)
 * [definemod](docs/definemod.md)
 * [reset](docs/reset.md)
-* [aux](docs/doc-aux.md)
-* [node](docs/doc-aux.md#node)
 * [alias](docs/alias.md)
+* [list of ease functions](docs/eases.md)
+
 </div>
-<div style="flex:50%" markdown="1">
+<div style="flex:75%" markdown="1">
 
-* [extra players](docs/players.md)
-* mode='end'
+* [player-specific](docs/players.md)
 * [flip](docs/flip.md)
 * [get](docs/get.md)
+* [aux](docs/doc-aux.md)
+  * [node](docs/doc-aux.md#node)
 * [func](docs/func.md)
-* [function eases](docs/func.md#func_for_function_eases)
-* [poptions](docs/func.md#poptions)
-* [aft](docs/aft.md)
-* shaders
-* [list of mods](docs/mods.md)
-* [list of ease functions](docs/eases.md)
+  * [function eases](docs/func.md#func_for_function_eases)
+  * [poptions](docs/func.md#poptions)
+* mode='end'
+
 </div>
 </div>
 
-Bonus:
-* [learn Lua](https://www.lua.org/manual/5.1/)
+### NotITG
 * [gradients](docs/gradients.md)
 * [splines](docs/splines.md)
+* [AFTs](docs/aft.md)
+* shaders
+* extra players
+* drawfuncs
+* actors
+* [list of mods](docs/mods.md)
 
-Example code:
-```lua
--- turn on invert
-ease {0, 1, outExpo, 100, 'invert'}
--- turn off invert
-ease {7, 1, outExpo, 0, 'invert'}
-```
 
+## Lua Tutorials
+* [for loop](docs/for.md)
+* [learn Lua](https://www.lua.org/manual/5.1/)
```

Comment:
I think this could be in its own section of "additional resources"

Reply:
ya true